data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -1319,14 +1319,513 @@ Marksman attacks are only affected if the chance to hit is greater than 60%."
     [/event]
 #enddef
 
+# AMLA abilities
+
+#define ABILITY_OPPORTUNIST
+    [dummy]
+        id = opportunist
+        name = _"opportunist"
+        description= _ "This ability makes Kaleh attack enemies with single strike of bolas when they leave his zone of control."
+    [/dummy]
+#enddef
+
+#define ABILITY_OPPORTUNIST_EVENTS
+    [event]
+        name = exit hex
+
+        id = opportunist
+        first_time_only = no
+
+        [filter] 
+            [filter_adjacent]
+                ability = opportunist
+                is_enemy = yes
+            [/filter_adjacent]
+        [/filter]    
+
+        [store_unit]
+            [filter]
+                find_in = unit
+            [/filter]
+            variable = opportunist_hit_units
+        [/store_unit]
+        [foreach]
+            array=opportunist_hit_units
+            variable=opportunist_hit_unit
+            [do]
+                [set_variable]
+                    name = opportunist_hit_roll
+                    rand = 1..100
+                [/set_variable]
+                [store_unit_defense]
+                    find_in = opportunist_hit_unit
+                    variable = opportunist_hit_chance
+                [/store_unit_defense]
+                [if]
+                    [variable]
+                        name = opportunist_hit_roll
+                        less_than_equal_to = $opportunist_hit_chance
+                    [/variable]
+                    [then]
+                        [harm_unit]
+                            [filter]
+                                find_in = opportunist_hit_unit
+                            [/filter]
+                            [filter_second]
+                                id = Kaleh
+                            [/filter_second]
+                            amount = 6
+                            damage_type = impact
+                            kill = yes
+                            experience = yes
+                            fire_event = yes
+                            slowed = yes
+                            animate = yes
+                            [primary_attack]
+                                name = bolas
+                            [/primary_attack]
+                        [/harm_unit]
+                    [/then]
+                [/if]
+            [/do]
+        [/foreach]
+
+        {CLEAR_VARIABLE opportunist_hit_units,opportunist_hit_chance,opportunist_hit_roll}
+    [/event]
+#enddef
+
+#define WEAPON_SPECIAL_TAUNT
+    [dummy]
+        id = taunt
+        name = _"taunt"
+        description= _ "It taunts the enemy making it try to attack Kaleh no matter what."
```

Comment:
The pronoun "it" appears twice here, applying to different things.

Reply:
Not to mention the initial "it" looks superfluous to me...

Reply:
I agree, you can just drop the initial "it". That said, the second "it" should maybe be "them" since it refers to a person.